src/org/rust/lang/core/psi/impl/mixin/RustStructItemImplMixin.kt

```diff
@@ -0,0 +1,21 @@
+package org.rust.lang.core.psi.impl.mixin
+
+import com.intellij.lang.ASTNode
+import com.intellij.openapi.util.Iconable
+import org.rust.lang.core.psi.RustStructItem
+import org.rust.lang.core.psi.impl.RustItemImpl
+import org.rust.lang.icons.RustIcons
+import org.rust.lang.icons.addVisibilityIcon
+import javax.swing.Icon
+
+/**
+ * Created by atsky on 03/11/15.
```

Comment:
Our code style says that such comments should not be included. 

https://github.com/alexeykudinkin/intellij-rust/blob/master/CONTRIBUTING.md#code-style

I suggest merging it as is, and making a separate cleanup pass later: we have similar comments elsewhere.